dem/getting-started/system-view.md

```diff
@@ -0,0 +1,29 @@
+---
+id: system-view
+title: System View
+---
+
+# System View
+
+This section allows you to analyze the overall health of the platform hosting your web application. To access it, open the main menu and select **System**.
+
+![image](../assets/getting-started/system-view-1.png)
+
+Prerequisites:
+
+- a Quanta license of type **OPS**, **Full**, or **Enterprise**.
+- a hosting platform that allows installing Linux packages — this excludes fully managed SaaS platforms like Salesforce or Shopify. For such services, the available platform visibility is not relevant because it is the SaaS provider’s responsibility to keep the application they provide running.
```

Comment:
```For such services, the available platform visibility is not relevant because it is the SaaS provider’s responsibility to keep the application they provide running.``` -> ```For these kind of hosting solutions, the system section would not be as relevant as you don't have control on the underlying platform as it's the SaaS provider responsibility to keep their application running smoothly```